### PR TITLE
Creator html project

### PR DESCRIPTION
**Pull request by azul100**
Creator  html

### CODE REVIEW

**Comment by azul100** [comment]
<!DOCTYPE html>
<html>

  <head>
  	<title>FireChat</title>
  	<meta name="viewport" content="width=device-width, initial-scale=1">
  	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/bootswatch/3.3.4/lumen/bootstrap.min.css">
  	<link rel="stylesheet" href="https://maxcdn.bootstrapcdn.com/font-awesome/4.3.0/css/font-awesome.min.css">
  	<script src="https://ajax.googleapis.com/ajax/libs/jquery/2.1.4/jquery.min.js"></script>
  	<script src="https://maxcdn.bootstrapcdn.com/bootstrap/3.3.4/js/bootstrap.min.js"></script>

  	<script src='https://cdn.firebase.com/js/client/2.2.1/firebase.js'></script>
  	<link rel="stylesheet" href="CloudChat.css">
  </head>

  <body>
  	<div class="container">
  		<div class="navbar navbar-default">
	      <div class="animsition" data-animsition-in="fade-in">
	        <button type="button" class="navbar-toggle" data-toggle="collapse" data-target=".navbar-collapse">
	          <span class="icon-bar"></span>
	          <span class="icon-bar"></span>
	          <span class="icon-bar"></span>
	        </button>
	        <a href="http://decoding.ghost.io/" class="navbar-brand">My Blog</a>
	        <a href="https://github.com/azul100"><i class="fa fa-github fa-2x navbar-brand hidden-xs" title="Github"></i></a>
	        <a href="https://www.linkedin.com/pub/zouggaghhamid"><i class="fa fa-linkedin-square fa-2x navbar-brand hidden-xs" title="LinkedIn"></i></a>
	        <a href="https://twitter.com/Fade_2_bliss"><i class="fa fa-twitter fa-2x navbar-brand hidden-xs" title="Twitter"></i></a>

	        
	        <div class="collapse navbar-collapse">
	          <div class="navbar-right"  id="navbar-spy">
	            <ul class="nav navbar-nav">
	              <li class="active"><a href="http://luckycode.org/"><i class="fa fa-home"></i> Home</a></li>

	              <li><a href="#" class="dropdown-toggle" data-toggle="dropdown"><i class="fa fa-link"></i> Links<b class="caret"></b></a>
	                <ul class="dropdown-menu">
	                  <li><a href="https://github.com/azul100"><i class="fa fa-github"></i> Github</a></li>
	                  <li><a href="https://www.linkedin.com/pub/zouggaghhamid"><i class="fa fa-linkedin-zouggaghhamid"></i> Linkedin</a></li>
	                  <li><a href="https://twitter.com/fdsq1975"><i class="fa fa-twitter"></i> Twitter</a></li>  
	                </ul>
	              </li>

	              
	              <li id="email"  data-toggle="tooltip" data-placement="left" title="sont85@gmail.com">
	                <a href="mailto:sont85@gmail.com?subject=Inquiry">
	                  <i class="fa fa-info"></i> Email
	                </a>
	              </li>

	            </ul>
	          </div>
	        </div> 
	      </div>
	    </div>
  		
	  	<div class="pull-right">
				<button type="button" class="btn btn-primary sign-in" data-toggle="modal" data-target="#logInModal">Log In</button>
				<button type="button" class="btn btn-info sign-in" data-toggle="modal" data-target="#signUpModal">Sign Up</button>
				<button id="annoymousButton" class="btn btn-warning sign-in">Annoymous</button>
				<button id="logOutButton" class="btn btn-danger hidden">Log Out</button>
			</div>


	  	<div id="header" class="text-center">
	  		<h2>FireChat</h2>
	  	</div>

			
			<div class="modal fade" id="logInModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
			  <div class="modal-dialog">
			    <div class="modal-content">
			      <div class="modal-header">
			        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
			        <h4 class="modal-title" id="myModalLabel" class="text-center">Log in</h4>
			      </div>
			      <div class="modal-body">
			        <div>
					  		<input class="form-control" type="email" id="logInEmail" placeholder="Email">
					  		<input class="form-control" type="password" id="logInPassword" placeholder="Password">
					  		<button id="logInButton" class="btn btn-lg btn-success" data-dismiss="modal">Log In</button>
					  	</div>
			      </div>
			    </div>
			  </div>
			</div>


			
			<div class="modal fade" id="signUpModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel" aria-hidden="true">
			  <div class="modal-dialog">
			    <div class="modal-content">
			      <div class="modal-header">
			        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
			        <h4 class="modal-title" id="myModalLabel" class="text-center">Sign Up</h4>
			      </div>
			      <div class="modal-body">
			        <div>
					  		<input class="form-control" type="email" id="signUpEmail" placeholder="Email">
					  		<input class="form-control" type="password" id="signUpPassword" placeholder="Password">
					  		<button id="createNewUserButton" class="btn btn-lg btn-info" data-dismiss="modal">Create User</button>
					  	</div>
			      </div>
			    </div>
			  </div>
			</div>

	  	<div id="template" class="individualMessage hidden">
	  		<span class="user"></span>
	  		<span class="message"></span>
	  		<span class="timeStamp pull-right"></span>
	  	</div>
	  	<div>
	  		<label for="userName">Chatroom Name: </label>
	  		<input type="text" id="userName" placeholder="Type Username">
	  	</div>
	  	<div class="input-group">
	  		<input class="form-control" type="text" id="textInput" placeholder="Enter Message">
	  		<span class="input-group-btn">
	  			<button class="btn btn-success" id="submitButton">Submit</button>
	  		</span>
	  	</div>
	  	<div id="messageBox">
	  	</div>

  	</div>

	<script src="CloudChat.js"></script>
  </body>

</html>
